rust/feature-flags/tests/test_flags.rs

```diff
@@ -6189,3 +6189,142 @@ async fn test_cohort_with_and_negated_cohort_condition() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_date_string_property_matching_with_is_date_after() -> Result<()> {
```

Comment:
NB: this test never failed; but I used it to verify that the Rust flag matching (that powers the `/flags` endpoint) was still working as expected.